### DISCUSSION TITLE
Welcome to BSC Discussions!

### DISCUSSION THREAD

**Discussion by shining-man** [shining-man/bsc_fw#2] [Announcements]
# 👋 Welcome!
We’re using Discussions as a place to connect with other members. We hope that you:
* Ask questions you’re wondering about.
* Share ideas.
* Engage with other members.
* Welcome others and are open-minded 💪.

**Comment by testpaul999** [comment]
Guten Tag,

ich würde mir auf allen Web-Sites einen „Home“-Button wünschen.
Am liebsten oben in der Satusleiste.
Dann muss Mann sich nicht immer „zurückklicken“.

danke

**Reply by shining-man** [reply]
Einfach ein Issue mit dem Wunsch aufmachen, macht es für mich einfacher.
